### PR TITLE
Add eslint fix

### PR DESCRIPTION
**Pull request by T1B0**
- update README.md
- add eslint 9
- fix lint issues

### CODE REVIEW

File: src/stream/consumer-stream.ts [hunk 1]
```diff
@@ -44,14 +44,15 @@ async function* genPoll(
 
 
 
-const singleConsumerStream = (config: ClientConfig) => async (
+export const singleConsumerStream = (config: ClientConfig) => async (
   poll: PollMessages,
-  interval: 1000
+  // interval: 1000
```
Comment (Tchoupinax) [inline src/stream/consumer-stream.ts:49]: Is it expected to keep the comment?

File: src/stream/consumer-stream.ts [hunk 2]
```diff
@@ -44,14 +44,15 @@ async function* genPoll(
...
 ): Promise<Readable> => {
   const c = await rawClientGetter(config);
   if (!c.isAuthenticated)
     await c.authenticate(config.credentials);
   const ps = Readable.from(genPoll(c, poll), { objectMode: true });
+  // const ps = Readable.from(genAutoCommitedPoll(c, poll, interval), { objectMode: true });
```
Comment (Tchoupinax) [inline src/stream/consumer-stream.ts:55]: Is it expected too?

**Comment by Tchoupinax** [comment]
Come code line are commented, if you agree it's expected, LGTM!

**Review by Tchoupinax — DISMISSED** [review]
Lines are commented, if you agree it's expected, LGTM!

File: src/stream/consumer-stream.ts (hunk 1 above)
Comment (T1B0) [reply src/stream/consumer-stream.ts:49]: yup it was unused and so raised a lint error, yet i still wanna play around that later probably

File: src/stream/consumer-stream.ts (hunk 2 above)
Comment (T1B0) [reply src/stream/consumer-stream.ts:55]: sorry i'm still playing r&d with those stream

**Comment by T1B0** [comment]
@Tchoupinax @spetz please note semantic release prefer squash or rebase merge, as merge commit just polute history uselessly :pray:

**Comment by spetz** [comment]
> @Tchoupinax @spetz please note semantic release prefer squash or rebase merge, as merge commit just polute history uselessly 🙏

Yes, I've only made it available to use squash merge anyway :)

**Review by spetz — APPROVED** [review]

**Pull request merged**